packages/cfd/src/features/Containers/financial-stp-real-account-signup.tsx

```diff
@@ -0,0 +1,242 @@
+import React from 'react';
+import { Div100vhContainer } from '@deriv/components';
+import { useIsAccountStatusPresent } from '@deriv/hooks';
+import { isDesktop, getAuthenticationStatusInfo } from '@deriv/shared';
+import { observer, useStore } from '@deriv/stores';
+import type { TCoreStores } from '@deriv/stores/types';
+import ProofOfAddress from '../Components/proof-of-address';
+import ProofOfIdentity from '../Components/proof-of-identity';
+import PersonalDetailsContainer from './personal-details-container';
+import { useCfdStore } from '../../Stores/Modules/CFD/Helpers/useCfdStores';
+import { JURISDICTION } from '../../Helpers/cfd-config';
+
+type TFinancialStpRealAccountSignupProps = {
+    onFinish: () => void;
+};
+
+type TNextStep = (index: number, value: { [key: string]: string | undefined }) => void;
+
+type TItem = {
+    refreshNotifications: TCoreStores['notifications']['refreshNotifications'];
+    removeNotificationMessage: TCoreStores['notifications']['removeNotificationMessage'];
+    removeNotificationByKey: TCoreStores['notifications']['removeNotificationByKey'];
+    addNotificationMessageByKey: TCoreStores['notifications']['addNotificationMessageByKey'];
+    authentication_status: TCoreStores['client']['authentication_status'];
+    account_settings: TCoreStores['client']['account_settings'];
+    email: TCoreStores['client']['email'];
+    is_fully_authenticated: TCoreStores['client']['is_fully_authenticated'];
+    landing_company: TCoreStores['client']['landing_company'];
+    residence_list: TCoreStores['client']['residence_list'];
+    states_list: TCoreStores['client']['states_list'];
+    fetchStatesList: TCoreStores['client']['fetchStatesList'];
+    account_status: TCoreStores['client']['account_status'];
+    jurisdiction_selected_shortcode: TCoreStores['modules']['cfd']['jurisdiction_selected_shortcode'];
+    has_submitted_cfd_personal_details: TCoreStores['modules']['cfd']['has_submitted_cfd_personal_details'];
+    onFinish: TFinancialStpRealAccountSignupProps['onFinish'];
+};
+
+type TItemsState<T extends TItem> = {
+    body: typeof ProofOfIdentity | typeof ProofOfAddress | typeof PersonalDetailsContainer;
+    form_value: { [key: string]: string | undefined };
+    forwarded_props: Array<Partial<keyof T>>;
+};
+
+const FinancialStpRealAccountSignup = observer(({ onFinish }: TFinancialStpRealAccountSignupProps) => {
```

Comment:
What is the purpose of this component? seems like the verifications are being handled here and there is no such thing as `financial stp` anymore 😅

Reply:
That's right Matin. The main purpose of renaming components was to change the old names which are not used anymore and also make the naming of components more understandable. Having `proof-of-address` as the name of component is much more understandable for future developer instead of having `poa` as its name.

Reply:
@shontzu-deriv kindly help to rename this file